google-cloud-spanner/clirr-ignored-differences.xml

```diff
@@ -192,6 +192,63 @@
     <className>com/google/cloud/spanner/StructReader</className>
     <method>java.util.List getPgJsonbList(java.lang.String)</method>
   </difference>
+  <!-- PG OID -->
+  <difference>
```

Comment:
It should not be necessary to add the internal methods to this file, if those methods are given a default implementation that just throws UnsupportedOperationException

Reply:
Done.

Reply:
These entries in `clirr-ignored-differences.xml` for `StructReader` can be removed, as we don't have those methods anymore.

Reply:
Done.